Tools-Override/FrameworkTargeting.targets

```diff
@@ -215,4 +215,114 @@
     </ItemGroup>
   </Target>
 
+  <!-- Binplacing targets -->
+  <Target Name="BinPlace"
+          DependsOnTargets="GetBinPlaceDirs;BinPlaceFiles;BinPlaceProps"
+          AfterTargets="CopyFilesToOutputDirectory"
+          Condition="'$(BinPlaceRef)' == 'true' OR '$(BinPlaceRuntime)' == 'true'" />
```

Comment:
It feels to me like BinPlaceRef and BinPlaceRuntime are a corefx concept and probably shouldn't be part of our common binplacing logic.